dask/bytes/core.py

```diff
@@ -19,8 +19,9 @@
 _open_text_files = dict()
 
 
-def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
-                     sample=True, compression=None, **kwargs):
+def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
+                     sample=True, compression=None, storage_options=None,
+                     **kwargs):
```

Comment:
Looking at all of the changes that result from this I'd definitely suggest just using kwargs.  I don't see a reason to keep a separate dictionary of options here any more.  I think it's idiomatic to just pass kwargs down.  We only did this with read_csv because we needed to pass some keywords to `read_csv` and the others to `read_bytes`.  At this stage that ambiguity is gone.